include/spdlog/details/spdlog_impl.h

```diff
@@ -31,6 +31,7 @@
 #include "../sinks/file_sinks.h"
 #include "../sinks/stdout_sinks.h"
 #include "../sinks/syslog_sink.h"
+#include "../sinks/android_sink.h"
```

Comment:
No. They are included because they are used in this file. If android is not used in spdlog_impl.h, it should not included